BuildAddOn.py

```diff
@@ -37,6 +37,11 @@ def PrepareParameters (args):
     elif platform.system () == 'Darwin':
         platformName = 'MAC'
 
+    # Load DevKit download data
+    devKitDataPath = pathlib.Path (__file__).absolute ().parent / 'APIDevKitLinks.json'
+    devKitDataFile = open (devKitDataPath)
+    devKitData = json.load (devKitDataFile)

```

Comment:
We should check the existence of that file before open, and raise an exception like in the case of the other config (I know the file is in the repository, but there may be local cases when it's missing or renamed).
 
Also I think there is no need for the existence of the _devKitDataFile_ and _configFile_ in a variable, the _open_ command should go inside the _json.load ()_ as parameter
```suggestion
    devKitData = json.load (open (devKitDataPath))

```
/

Reply:
The `open ()` function throws an exception if the file doesn't exist. In the case of the other config file, we check if the supplied path is a directory or not.